config/routes.rb

```diff
@@ -20,6 +20,8 @@
   end
 
   resources :users, only: [:new, :create, :edit, :update] do
+    resources :reservations, only: [:index, :show, :new, :create, :update]
+    resources :messages
```

Comment:
I'm not sure, I think @valerietrudell put these nested routes in a while back for user dashboard stuff.